website/content/docs/connect/config-entries/ingress-gateway.mdx

```diff
@@ -784,7 +784,11 @@ spec:
         {
           name: 'Port',
           type: 'int: 0',
-          description: 'The port that the listener should receive traffic on.',
+          description: `The port on which the listener should receive traffic.
```

Comment:
A hint that the interface/IP used here is derived from the interface/IP supplied to `-address` would also be helpful I think.

Reply:
I've modified this paragraph to note the port will be bound to the IP specified in `-address`.